cmd/go-getter/progress_tracking.go

```diff
@@ -10,8 +10,7 @@ import (
 )
 
 // DefaultProgressBar is the default instance of a cheggaaa
-// progress bar. It is recommended to use DefaultProgressBar
-// in all places.
+// progress bar.
 var DefaultProgressBar getter.ProgressTracker = &ProgressBar{}
```

Comment:
Almost there. Just need to make this unexported now and we're good!

Reply:
Okay, done 🙂